src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Navbar.css'
+
+export const Navbar = () => {
+  const getClassName = ({ isActive }: { isActive: boolean }): string => {
+    return isActive ? 'active-link' : ''
+  }
+  return (
+    <nav>
+      <a className={`link ${getClassName}`} href="/">
+        Home
+      </a>
+      <a className={`link ${getClassName}`} href="/counter">
```

Comment:
If we are using anchor tag with an application using React-router then we are not making it as a SPA. Use Link component from React-router isntead

Reply:
Done.